src/service/video-manager-impl.js

```diff
@@ -894,13 +945,235 @@ class VideoEntry {
     // Update docking state
     if (this.scrollMap_(DOCK_SCALE, 1) == DOCK_SCALE) {
       this.dockState_ = DockStates.DOCKED;
+      this.initializeDragging_();
```

Comment:
this is measuring, which we don't wanna do during a mutate, needs to go to its own `measure` block.